PetProject/Views/LoginScreen/LoginViewController.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  LoginViewController.swift
+//  PetProject
+//
+//  Created by Тимофей Лукашевич on 24.03.22.
+//
+
+import UIKit
+import Combine
+
+protocol LoginViewControllerDelegate: AnyObject, Coordinator {
+    func showMainScreen()
+}
+
+class LoginViewController: UIViewController {
+
+    // MARK: - Subviews
+    weak var delegate: LoginViewControllerDelegate?
+    var loginScreenVM: LoginViewModel?
```

Comment:
It is a contract. And without VM flow doesn't work and in this case you can do like this:
`var vm: LoginViewModel!`. 
Otherwise, be careful with "!"